debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleSchemaChangeEventEmitter.java

```diff
@@ -54,7 +54,7 @@ public void emitSchemaChangeEvent(Receiver receiver) throws InterruptedException
     private SchemaChangeEventType getSchemaChangeEventType() {
         switch(ddlLcr.getCommandType()) {
             case "CREATE TABLE": return SchemaChangeEventType.CREATE;
-            case "ALTER TABLE": throw new UnsupportedOperationException("ALTER TABLE not yet implemented");
+//            case "ALTER TABLE": throw new UnsupportedOperationException("ALTER TABLE not yet implemented");
```

Comment:
Is that intentional?

Reply:
@gunnarmorling This a is a good question - I need it for `ALTER TABLE ADD SUPPLEMENTAL LOG DATA (ALL) COLUMNS`. As I suppose this is soething that is needed we should either handle this case or throw only `WARN` for all alters

Reply:
Yes, at least a warning should be logged.